### ISSUE TITLE
Debugging ability using debugger keyword or breakpoint in the sources tab

### ISSUE THREAD

**Issue by anuragsinghbam** [codesandbox/codesandbox-client#8410]
# 🌈 Debugging Ability

Debugging ability using debugger keyword and breakpoint in the sources tab. The devtool is not able to load the source files that's why it doesn't stop on the debugger keyoword.

Here are some screenshots, for better understanding.
 
Chrome Sources Tab
<img width="1822" alt="Chrome Sources Tab" src="https://github.com/codesandbox/codesandbox-client/assets/61440053/d8ed2743-48e4-4552-b5cd-6f0cccfd875a">

Code SandBox Sources Tab
<img width="1822" alt="Code SandBox Sources Tab" src="https://github.com/codesandbox/codesandbox-client/assets/61440053/34246739-2c8b-444b-8f22-04cb57038d5c">

Chrome Element and Console Tab
<img width="1822" alt="Chrome Element and Console Tab" src="https://github.com/codesandbox/codesandbox-client/assets/61440053/0defe6a3-a745-4481-94ba-d0fcafa0264e">

Code SandBox Elements and Console Tab
<img width="1822" alt="Code SandBox Elements and Console Tab" src="https://github.com/codesandbox/codesandbox-client/assets/61440053/6312235d-30e3-413a-9e19-860401b13980">

**Comment by CompuIves** [comment]
Yep... Looks like a limitation in the chrome devtool connection. We can take a look later at this one!
